arches_lingo/src/arches_lingo/pages/SchemeReport.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import Splitter from "primevue/splitter";
+import SplitterPanel from "primevue/splitterpanel";
+import SchemeLicense from "@/arches_lingo/components/scheme/report/SchemeLicense.vue";
+import SchemeNote from "@/arches_lingo/components/scheme/report/SchemeNote.vue";
+import SchemeUri from "@/arches_lingo/components/scheme/report/SchemeUri.vue";
+import SchemeStandard from "@/arches_lingo/components/scheme/report/SchemeStandard.vue";
+import SchemeAuthority from "@/arches_lingo/components/scheme/report/SchemeAuthority.vue";
+</script>
+
+<template>
+    <div>
+        <Splitter>
```

Comment:
YAGNI the splitter. IMO don't build out a single-panel splitter, let's punt it till it's needed

Reply:
AH -- it's used in the follow-on PR to this. Resolving 👍